packages/bff/src/services/catalogService.ts

```diff
@@ -122,5 +123,41 @@ export function catalogServiceBuilder(
 
       return response;
     },
+    createEService: async (
+      eServiceSeed: bffApi.EServiceSeed,
+      { headers }: WithLogger<BffAppContext>
+    ): Promise<CreatedResource> => {
+      const { id } = await catalogProcessClient.createEService(eServiceSeed, {
```

Comment:
```suggestion
      const response = await catalogProcessClient.createEService(eServiceSeed, {
```

Then just return the `response` so you avoid destructuring and rebuilding the object

Reply:
Or you could just remove the `{ ... }` to directly return `catalogProcessClient.createEService(...` here and in the other added functions